### ISSUE TITLE
Create DID explainer

### ISSUE THREAD

**Issue by burnburn** [w3c/did-core#94]
This isn't actually a piece of work for the did-core spec directly, but at some point we will need review by the W3C Technical Architecture Group (TAG), and for any work they review they request information in a particular format referred to as an Explainer.

Work on an Explainer began in the CCG (https://github.com/w3c-ccg/community/issues/38), and it is now appropriate for this group to take over that work.

This issue can remain until we decide where best to track it.

**Comment by brentzundel** [comment]
could this be part of the [implementation guide?](https://github.com/w3c/did-imp-guide)

**Comment by burnburn** [comment]
No.  There is a very specific set of questions they want answers to, and only those.  This needs to be separate.  When the group is closer to being ready for an initial review I will be happy to restart this work.

**Comment by burnburn** [comment]
When the DID core spec is ready, we will need to follow the template described in [this document](https://w3ctag.github.io/explainers) and also complete the [Security and Privacy Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/) before we can ask for a TAG review.

**Comment by burnburn** [comment]
I had volunteered to do this.  Still plan on doing it but happy to have someone else take it on or help.

**Comment by brentzundel** [comment]
we have a plan, now we need to act. We will do so.

**Comment by wyc** [comment]
Interested in contributing content to this and also ensuring a smooth transition from CCG if desired. Please let me know the best way to engage if it would help.

**Comment by brentzundel** [comment]
DID Explainer now exists! 
https://github.com/w3c/did-wg/blob/master/did-explainer.md

**Comment by iherman** [comment]
Comments on the [explainer draft](https://github.com/w3c/did-wg/blob/master/did-explainer.md). I hope they are helpful.

## “Introduction” section

* [ ] “such as a distributed ledger or blockchain”: for most laypersons (like me) these two term are interchangeable on a higher level. (There may be some subtle differences, maybe the second is an implementation approach to the former, but that detail is not for this explainer.) I think the concept of a distributed ledger should be enough, and makes the idea clear without raising an anti-blockchain reaction.
    
    Also, having a different example for a very different DID "oracle" would be good here to avoid the impression that DID-s are _only_ a distributed ledger application and not a separate concept. We do have methods based on github, I believe we also have IPFS based methods, we even have methods based on twitter:-)
* [ ] “to-be-created DID Registry”: we should specifically point at our registry document, probably emphasizing that it is not final. It reinforces the value of the WG’s work.
 
## “Goals” section

We may want to add, among the non-goals, the fact that _browsers are not required to do anything with DIDs_. Of course, they may, just as there are extension for URI schemes like DOI-s, but that does not affect the core browser engines. (Such a remark is very important, otherwise browser vendors may be up in arms against the DID work!)

## “Detailed design discussion” section

* [ ] In general, I believe a “figure is worth a thousand words” applies here: I believe we should have a figure.
  
   Figure 1 in the spec is probably too detailed and spec oriented. I used a [simple diagram](https://docs.google.com/drawings/d/1VQFh6dKAPbemKIn69UJdgrQ81y8tRdSEBONArwjtdmM/edit) as part of my presentation; maybe a version thereof could work.

### “DID Documents” subsection

* [ ] “An example is the “key” method”: let us put a link to the method description itself.

### “DID Dereferencing” subsection

* [ ] This section comes a bit out of the blue here, and is not really understandable. Either add more about the topic or remove it. (I would propose to remove it. It is not a core feature.)

### “Considered alternatives” subsection

#### “Open ID/OIDC” sub-subsection

* [ ] “they are actually complementary to such technologies”: this should be elaborated upon. The referenced DIF medium article is for the initiated not for those who would want to understand the relationships...

#### “Other URI schemes” sub-subsection

The Use case document’s abstract has 4 essential characteristics of the type of identifiers that the use cases need:

1. decentralized
2. persistent
3. cryptographically verifiable
4. resolvable

I would actually add a 5th one:

5. ease of creation

The statement is that while existing URI schemes may fulfill one or more of these requirements, none, apart from DID-s, fulfill all of those. 

* [ ] I find the succinct terms from the Use Cases document better than the terms used in the explainer.

**Comment by brentzundel** [comment]
DID Explainer has been written and reviewed.
